### PR TITLE
Azure native   add link to azure docs, more

### PR DESCRIPTION
**Pull request by zuluecho9**
Azure native - add limitations and notes about Data Plus features that are unavialable.

### CODE REVIEW

File: src/content/docs/infrastructure/microsoft-azure-integrations/get-started/azure-native.mdx [hunk 1]
```diff
@@ -12,21 +12,36 @@ If your team deploys software and cloud services through Azure, our [Azure Nativ
...
-[Learn more about benefits on the Azure Marketplace.](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/newrelicinc1635200720692.newrelic_liftr_payg)
+### Available editions [#edition]
+
+Our Standard and Pro [editions](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#editions) are available, but not yet Enterprise.
+
+### Available Data Plus features [#data-plus]
+
+This service comes with our [Data Plus option](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing/#data-plus), but currently these features are **not** yet available: 
+
+* Historical exports
```
Comment (emoryzhao) [inline src/content/docs/infrastructure/microsoft-azure-integrations/get-started/azure-native.mdx:37]: Historical data export

File: src/content/docs/infrastructure/microsoft-azure-integrations/get-started/azure-native.mdx [hunk 2]
```diff
@@ -12,21 +12,36 @@ If your team deploys software and cloud services through Azure, our [Azure Nativ
...
+* Streaming data
```
Comment (emoryzhao) [inline src/content/docs/infrastructure/microsoft-azure-integrations/get-started/azure-native.mdx:38]: Streaming data export

File: src/content/docs/infrastructure/microsoft-azure-integrations/get-started/azure-native.mdx [hunk 3]
```diff
@@ -12,21 +12,36 @@ If your team deploys software and cloud services through Azure, our [Azure Nativ
...
+* Vulnerability management
```
Comment (emoryzhao) [inline src/content/docs/infrastructure/microsoft-azure-integrations/get-started/azure-native.mdx:39]: Vulnerability Management

File: src/content/docs/infrastructure/microsoft-azure-integrations/get-started/azure-native.mdx [hunk 4]
```diff
@@ -12,21 +12,36 @@ If your team deploys software and cloud services through Azure, our [Azure Nativ
...
+Our Standard and Pro [editions](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#editions) are available, but not yet Enterprise.
+
+### Available Data Plus features [#data-plus]
+
+This service comes with our [Data Plus option](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing/#data-plus), but currently these features are **not** yet available: 
+
+* Historical exports
+* Streaming data
+* Vulnerability management 
+* FedRAMP and HIPAA (due to Enterprise edition not yet being available)
```
Comment (emoryzhao) [inline src/content/docs/infrastructure/microsoft-azure-integrations/get-started/azure-native.mdx:40]: FEDRamp Moderate and HIPAA (HITrust Certified)

**Review by emoryzhao — CHANGES_REQUESTED** [review]

File: src/content/docs/infrastructure/microsoft-azure-integrations/get-started/azure-native.mdx [hunk 5]
```diff
@@ -12,21 +12,34 @@ If your team deploys software and cloud services through Azure, our [Azure Nativ
 * Deploy New Relic APM and infrastructure agents through an automated setup flow 
 * Manage the metrics and logs for all your resources through an interactive dashboard
 
-## What you can monitor [#features]
+## Features [#features]
 
-You can deploy the following agents through the Azure Native New Relic service:
+The features and integrations available for the Azure Native New Relic Service is different from regular, non-Azure New Relic.
```
Comment (emoryzhao) [inline src/content/docs/infrastructure/microsoft-azure-integrations/get-started/azure-native.mdx:17]: You can deploy the following agents through the Azure Native New Relic Service:

File: src/content/docs/infrastructure/microsoft-azure-integrations/get-started/azure-native.mdx [hunk 6]
```diff
@@ -12,21 +12,34 @@ If your team deploys software and cloud services through Azure, our [Azure Nativ
...
+**APM agents:**
 
 * .NET
 * Java
 
-Infrastructure agents: 
+**Infrastructure agent for:** 
 
 * Windows
 * Linux
```
Comment (emoryzhao) [inline src/content/docs/infrastructure/microsoft-azure-integrations/get-started/azure-native.mdx:30]: Other agents and integrations are available through the New Relic platform and can be found through the [I/O integrations](https://newrelic.com/instant-observability/).

File: src/content/docs/infrastructure/microsoft-azure-integrations/get-started/azure-native.mdx [hunk 7]
```diff
@@ -12,21 +12,34 @@ If your team deploys software and cloud services through Azure, our [Azure Nativ
...
 
 * Windows
 * Linux 
 
-[Learn more about benefits on the Azure Marketplace.](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/newrelicinc1635200720692.newrelic_liftr_payg)
+### Unavailable features [#data-plus]
+
+Our Standard and Pro [editions](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#editions) are available, but not yet Enterprise.
+
+This service comes with our [Data Plus option](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing/#data-plus), but currently these Data Plus features are **not** yet available:
```
Comment (emoryzhao) [inline src/content/docs/infrastructure/microsoft-azure-integrations/get-started/azure-native.mdx:35]: This service comes with our [Data Plus option](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing/#data-plus). Not all Data Plus features are available to Azure Native New Relic Service subscribers, please see the Data Plus [documentation](https://docs.newrelic.com/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing/#data-plus).

File: src/content/docs/infrastructure/microsoft-azure-integrations/get-started/azure-native.mdx [hunk 8]
```diff
@@ -12,21 +12,34 @@ If your team deploys software and cloud services through Azure, our [Azure Nativ
...
 * Linux 
 
-[Learn more about benefits on the Azure Marketplace.](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/newrelicinc1635200720692.newrelic_liftr_payg)
+### Unavailable features [#data-plus]
+
+Our Standard and Pro [editions](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#editions) are available, but not yet Enterprise.
+
+This service comes with our [Data Plus option](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing/#data-plus), but currently these Data Plus features are **not** yet available: 
+
+* Historical data export
```
Comment (emoryzhao) [inline src/content/docs/infrastructure/microsoft-azure-integrations/get-started/azure-native.mdx:37]: Remove Lines 37-41

**Review by emoryzhao — CHANGES_REQUESTED** [review]

**Review by bradleycamacho — APPROVED** [review]

**Pull request merged**